zgw_consumers/test/factories.py

```diff
@@ -0,0 +1,38 @@
+import factory
+
+from zgw_consumers.models import Service
+
+
+class UriPathFaker(factory.Faker):
+    def __init__(self, **kwargs):
+        super().__init__("uri_path", **kwargs)
+
+    def generate(self, extra_kwargs=None):
+        uri_path = super().generate(extra_kwargs)
+        # faker generates them without trailing slash, but let's make sure this stays true
+        # zgw_consumers.Service normalizes api_root to append missing trailing slashes
+        assert not uri_path.endswith("/")
+        return f"{uri_path}/"
+
+
+class ServiceFactory(factory.django.DjangoModelFactory):
+    label = factory.Sequence(lambda n: f"API-{n}")
+    api_root = UriPathFaker()  # FIXME: this should be a fully qualified URL
```

Comment:
this one needs fixing before we merge this - api_root is URL validated so it must have a protocol and netloc

Reply:
https://github.com/maykinmedia/zgw-consumers/pull/73/commits/b5114610c57e64e8b0f6f6e9b20ff5a9d3a8e12c
Is this what you expected?